AUTHORS

```diff
@@ -868,5 +868,132 @@ Super Zheng <superzheng@tencent.com>
 Rafał Pocztarski <r.pocztarski@gmail.com>
 Michael Ruddy <mruddybtc@gmail.com>
 Andy Bettisworth <andy.bettisworth@accreu.com>
+Jörg Krause <joerg.krause@embedded.rocks>
+Alexander Martin <alex@suitupalex.com>
+Prince J Wesley <princejohnwesley@gmail.com>
+janriemer <janriemer@users.noreply.github.com>
+Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
+Martin von Gagern <Martin.vGagern@gmx.net>
+Hideki Yamamura <hideki.yamamura@gmail.com>
+Lenny Markus <lmarkus@paypal.com>
+Nelson Pecora <nelson@yoshokatana.com>
+ronkorving <ron@ronkorving.nl>
+Xotic750 <xotic750@gmail>
+Nicholas Young <nicholas@nicholaswyoung.com>
+chrisjohn404 <chris.s.johnson4@gmail.com>
+Bo Borgerson <gigabo@gmail.com>
+Mudit Ameta <zeusdeux@gmail.com>
+Adri Van Houdt <adri.van.houdt@gmail.com>
+Max Barinov <me@maxbarinov.com>
+Damian Schenkelman <damian.schenkelman@gmail.com>
+iamchenxin <iamchenxin@gmail.com>
+Nik Nyby <nnyby@columbia.edu>
+Ben Ripkens <bripkens.dev@gmail.com>
+Ryan Sobol <contact@ryansobol.com>
+jona <jona@jona.io>
+Anna Henningsen <sqrt@entless.org>
```

Comment:
@jasnell You can remove this line, I feel fine being listed once, but thanks. :smile: